go.mod

```diff
@@ -6,13 +6,18 @@ require (
 	github.com/dustin/go-humanize v1.0.0
 	github.com/openebs/api/v2 v2.3.0
 	github.com/openebs/jiva-operator v1.12.2-0.20210607114402-811a3af7c34a
+	github.com/openebs/zfs-localpv v1.8.0
 	github.com/pkg/errors v0.9.1
 	github.com/ryanuber/columnize v2.1.0+incompatible
 	github.com/spf13/cobra v1.1.1
 	github.com/spf13/viper v1.7.0
-	k8s.io/api v0.21.0
-	k8s.io/apimachinery v0.21.0
-	k8s.io/cli-runtime v0.21.0
-	k8s.io/client-go v0.21.0
+	golang.org/x/crypto v0.0.0-20201221181555-eec23a3978ad // indirect
+	golang.org/x/term v0.0.0-20201126162022-7de9c90e9dd1 // indirect
```

Comment:
These two dependencies might get trimmed with `go mod tidy`, please check once.

Reply:
Please bump up your golang compiler to `1.16`, this will likely solve the go.mod & go.sum out-of-sync issue.